back/app/policies/authoring_assistance_response_policy.rb

```diff
@@ -0,0 +1,5 @@
+class AuthoringAssistanceResponsePolicy < ApplicationPolicy
+  def create?
+    user&.active? && user.super_admin?
```

Comment:
As a prototype is it also worth checking the feature flag here on the backend, so we know it's only ever invoked on platforms we want it to be? Too much maybe?